modules/web/src/main/ui/FaqUi.scala

```diff
@@ -391,5 +391,21 @@ final class FaqUi(helpers: Helpers, sitePages: SitePages)(
               ),
               "."
             )
+          ),
+          question(
+            "stop-chess-addiction",
+            trf.stopMyselfFromPlaying.txt(),
+            p(
+              trf.adviceOnMitigatingAddiction(
+                a(href := "https://getcoldturkey.com")("ColdTurkey"),
+                a(href := "https://freedom.to")("Freedom"),
+                a(href := "https://www.proginosko.com/leechblock")("LeechBlock"),
+                a(href := "https://github.com/ornicar/userstyles/blob/master/lichess.fewer-pools.user.css")(
+                  "this one"
+                ),
+                a(href := "https://lichess.org/page/userstyles")("here"),
```

Comment:
please don't use "here" as a link text, but instead use what the link is about.

`you might also be interested in <a href>Lichess userstyles</a>`